src/Microsoft.AspNetCore.Rewrite/Internal/UrlActions/RedirectAction.cs

```diff
@@ -9,25 +10,33 @@ namespace Microsoft.AspNetCore.Rewrite.Internal.UrlActions
     public class RedirectAction : UrlAction
     {
         public int StatusCode { get; }
```

Comment:
Members order: we put class members in following order
1. fields, statics, consts
2. constructors
3. properties
4. publish methods
5. private methods